static/preloads/handleErrors.js

```diff
@@ -0,0 +1,31 @@
+const electron = require('electron');
+const path = require('path');
+const ejs = require('ejs');
+const fs = require('fs');
+const { URL } = require('whatwg-url');
+
+const { ipcRenderer } = electron;
+
+ejs.fileLoader = (filePath) => {
+  return fs.readFileSync(path.normalize(path.join(__dirname, '..', filePath)));
+};
+
+module.exports = function handleErrors(document) {
+  function renderErrorPage(data) {
+    ejs.renderFile('error.ejs', data, (err, content) => {
+      if (err) {
+        // TODO
```

Comment:
TODO?

Reply:
Yeah, I'm not really sure what to do in this case.  Seems like a scenario that we shouldn't ever run into.  Maybe I'll just add a `console.error` call for now.

Reply:
@DalderupMaurice Updated.